artifactory/cli.go

```diff
@@ -1374,6 +1375,24 @@ func dotnetCmd(c *cli.Context) error {
 	return commands.Exec(dotnetCmd)
 }
 
+func getNugetAndDotnetConfigFields(configFilePath string) (rtDetails *config.ArtifactoryDetails, targetRepo string, useNugetV2 bool, err error) {
```

Comment:
I noticed that only for the nuget and dotnet commands, the config is read here, and not in jfrog-cli-core. All other commands use the ```SetConfigFilePath``` API when creating the command.
What is the reason this has been now modified for nuget and dotnet? I'm not sure which method is better. My concern is more around consistency between all c ommands.

Reply:
The legacy nuget command doesn't use a config file so it doesn't really make sense to read it in the core command.